AuthenticationApp/Controller/SignUpVC.swift

```diff
@@ -21,12 +22,18 @@ class SignUpVC: UIViewController {
     @IBOutlet weak var loginHintLabel: UILabel!
     @IBOutlet weak var signUpButton: UIButton!
     @IBOutlet weak var loginButton: UIButton!
+    var imagePicker = UIImagePickerController()
```

Comment:
This var is only used in the `presentImagePicker` function.

Reply:
Done

Reply:
Done